Cargo.toml

```
@@ -44,7 +45,8 @@ electrum = ["bdk/electrum"]
 compact_filters = ["bdk/compact_filters"]
 esplora = []
 esplora-ureq = ["esplora", "bdk/use-esplora-ureq"]
-esplora-reqwest = ["esplora", "bdk/use-esplora-reqwest"]
+async-interface = ["bdk/async-interface"]
+esplora-reqwest = ["esplora", "bdk/use-esplora-reqwest", "async-interface"]
```

Ah found the problem.. We need to also enable `default-tls` for request.. 
```suggestion
esplora-reqwest = ["esplora", "bdk/use-esplora-reqwest", "bdk/reqwest-default-tls", "async-interface"]
```